Update EIP-1: add Network Upgrade Inclusion (#10936)
ethereum/EIPs: EIPS/eip-1.md

These calls generally result in a "rough consensus" around what EIPs should be implemented. This "rough consensus" rests on the assumptions that EIPs are not contentious enough to cause a network split and that they are technically sound.

### Network Upgrade Inclusion

> EIPs targeting a network upgrade must be proposed for inclusion in that upgrade's Meta EIP. The inclusion stages—from "Proposed for Inclusion" through "Scheduled for Inclusion" to "Included"—are defined in [EIP-7723](./eip-7723.md). The scope for each upgrade is finalized within a few weeks of the previous fork going live on mainnet; after this deadline, only scheduled EIPs and security-critical proposals are considered. Proposals do not carry over between upgrades—authors must re-propose their EIP for each upgrade cycle.

Reviewer: ```suggestion
EIPs targeting a network upgrade must be proposed for inclusion in that upgrade's Meta EIP. The inclusion stages—from "Proposed for Inclusion" through "Scheduled for Inclusion" to "Included"—are defined in [EIP-7723](./eip-7723.md). Once the scope for each upgrade is finalized (generally within a few weeks of the previous upgrade going live on mainnet), only security-critical new proposals will be considered. Proposals do not carry over between upgrades—authors must re-propose their EIP for each upgrade cycle.
```
Allow for more flexibility on scope timing
